mmv1/third_party/terraform/go.mod.erb

```diff
@@ -22,11 +22,9 @@ require (
 	github.com/spf13/afero v1.2.2 // indirect
 	github.com/zclconf/go-cty v1.5.1 // indirect
 	golang.org/x/net v0.0.0-20210503060351-7fd8e65b6420
-	golang.org/x/oauth2 v0.0.0-20210514164344-f6687ab2804c
-	golang.org/x/sys v0.0.0-20210611083646-a4fc73990273 // indirect
-	golang.org/x/tools v0.1.3 // indirect
-	google.golang.org/api v0.48.0
-	google.golang.org/grpc v1.38.0
+	golang.org/x/oauth2 v0.0.0-20210819190943-2bc19b11175f
```

Comment:
Can we make the go.mod/go.sum changes in a separate CL? They seem to be breaking a seemingly unrelated file, and it'll be easier to debug that in isolation.